lattices/LatticeMath/LatticeStatistics.tcc

```diff
@@ -875,13 +872,13 @@ Bool LatticeStatistics<T>::generateStorageLattice() {
        os_p << LogIO::NORMAL1
             << "Creating new statistics storage lattice of shape " << storeLatticeShape << endl << LogIO::POST;
     }
-    pStoreLattice_p = new TempLattice<AccumType>(
+    pStoreLattice_p.reset (new TempLattice<AccumType>(
         TiledShape(storeLatticeShape,
         tileShape), useMemory
-    );
+    ));
```

Comment:
My preference would be to group things within "()" on the same line, unless that line exceeds 80 characters in length, in which case, add lines at "(", so
`
pStoreLattice_p.reset(
    new TempLattice<AccumType(
        TiledShape(storeLatticeShape, tileShape), useMemory
    )
);
`

Reply:
done